lib-ng/scanners/pathsense.js

```diff
@@ -23,7 +23,15 @@ const stopMonitoring = () => {
     PathSense.stopMonitoringAllRegions();
 };
 
+/**
+* Sort geofence-regions by distance inside a defined radius from a predefined location.
+*/
+const sortRegionsByDistance = (regions, location, defaultRadius = 5000) => {
```

Comment:
Providing a default value for `radius` can lead to strange bugs when not specifying it. As a developer using this method, I would expect to have the regions sorted. When not providing a value I might get less regions back.

Didn't you already rename `sortRegionsByDistance` to `getNearestRegions`?